cmdeploy/src/cmdeploy/__init__.py

```diff
@@ -748,6 +748,20 @@ def deploy_chatmail(config_path: Path, disable_mail: bool) -> None:
     _remove_rspamd()
     opendkim_need_restart = _configure_opendkim(mail_domain, "opendkim")
 
+    #
+    # If this system is pre-populated with data from a previous instance,
+    # we might need to adjust ownership of files.
+    #
+    stateful_paths = {
+        "/etc/dkimkeys": "opendkim",
+        "/home/vmail/mail": "vmail",
```

Comment:
I just realized - `/home/vmail/mail` can be *very* large. On large instances, chowning it can take several minutes; not something we should call on every `cmdeploy run`.

I suggest that we only call the chown steps when `--disable-mail` was passed? Then they will not be executed during *every* deploy.